### ISSUE TITLE
FEATURE

### ISSUE THREAD

**Issue by sg16bbrave** [Musish/Musish#533]
Is it possible to add the new Apple Music sound quality features? I mean, lossless and Hi-Res lossless mainly.

**Comment by macecchi** [comment]
Spatial Audio would be great too.

It looks like the new quality options are not available on the Music API yet though: https://developer.apple.com/documentation/musickitjs/musickit/playbackbitrate 😕

**Comment by Nirzak** [comment]
> Spatial Audio would be great too.
> 
> It looks like the new quality options are not available on the Music API yet though: https://developer.apple.com/documentation/musickitjs/musickit/playbackbitrate 😕

Yup it's not available till now. But I think it will be also available on the API as the API is also used by apple music web app.

**Comment by dumbuz** [comment]
If it's been this long without anything, I guess not.

**Comment by Lucyy-cloud** [comment]
Yes, ViWizard Audio Converter allows you to customize the output audio files, such as changing the volume, speed, pitch, set the codec, channel, sample rate, and bit rate. Furthermore, it can convert the all audio files at a 30× faster speed while at the same time maintaining high quality for the output files.
